requirements.txt

```diff
@@ -0,0 +1,2 @@
+tomlkit
+pypandoc_binary
```

Comment:
Please pin these to specific versions.
(~= is probably the right operator in this situation).